Lib/defconQt/spaceCenter.py

```diff
@@ -450,12 +488,13 @@ def mousePressEvent(self, event):
         # XXX: shouldnt set focus if we are in input field...
         self.setFocus(Qt.MouseFocusReason)
         if event.button() == Qt.LeftButton:
-            if self._verticalFlip:
-                baselineShift = -self.descender
-            else:
-                baselineShift = self.ascender
+            # if self._verticalFlip:
+            #     baselineShift = -self.descender
+            # else:
+            #     baselineShift = self.ascender  # unused
```

Comment:
Remove the unused comment and put a FIXME on top of the comment block that this should be investigated.